contracts/FeeDisbursal/FeeDisbursal.sol

```diff
@@ -0,0 +1,269 @@
+pragma solidity 0.4.25;
+
+import "../Universe.sol";
+import "../AccessControl/AccessControlled.sol";
+import "../Reclaimable.sol";
+import "../Standards/IFeeDisbursal.sol";
+import "../Identity/IIdentityRegistry.sol";
+import "../SnapshotToken/BasicSnapshotToken.sol";
+import "../Serialization.sol";
+import "../Math.sol";
+import "../Standards/IERC223Token.sol";
+
+
+contract FeeDisbursal is
+    AccessControlled,
+    Reclaimable,
+    IFeeDisbursal,
+    IdentityRecord,
+    Serialization,
+    Math
+{
+
+    uint256 constant UINT256_MAX = 2**256 - 1;
```

Comment:
make section for constants. I think it goes after types ;>

Reply:
:) Done